src/plot_api/plot_api.js

```diff
@@ -178,11 +178,15 @@ function plot(gd, data, layout, config) {
         gd.calcdata[i][0].trace = gd._fullData[i];
     }
 
+    function isHidden(gd) {
+        var display = window.getComputedStyle(gd).display;
+        return !display || display === 'none';
+    }
```

Comment:
Isn't this the same helpers as in `Plots.resize`? 

Can we make this a `Lib` function?

Reply:
Yes, good idea!

Done in 913bb6877df1e7668748ad4b3e535e38bbf25e6b